### ISSUE TITLE
Init pos or color save

### ISSUE THREAD

**Issue by tombed631** [skydoves/ColorPickerView#19]
ColorPickerReferenence does not have Sliders (what i need is BrightnessSlider) so can you add to this projekct initPosition methods? I see only in init() setCenter() call... But it would be needed setter with params to set initial position...

I need both of saving colors and brithnessSlider. Im looking forward for your reply.

**Comment by dtboy1995** [comment]
https://github.com/skydoves/ColorPickerView/issues/8

**Comment by skydoves** [comment]
@tombed631 @dtboy1995 
Hi, sorry for late reply.
It's released v2.1.0
You can save & restore all of the `ColorPickerView` data like this :

```java
colorPickerView.setPreferenceName("MainColorPickerView"); // sets the preference name.  This restores the stored date.
colorPickerView.setLifecycleOwner(this); // saves all of the ColorpickerView data when the lifecycleOwner's onDestroy method called automatically.
```

or you can save data manually
```java
ColorPickerPreferenceManager.getInstance(this).saveColorPickerData(colorPickerView);
```

Thank you!
